gnocchi/tests/base.py

```diff
@@ -390,6 +390,7 @@ def _create_metric(self, archive_policy_name="low"):
     def trigger_processing(self, metrics=None):
         if metrics is None:
             self.chef.process_new_measures_for_sack(
-                self.incoming.sack_for_metric(self.metric.id), sync=True)
+                self.incoming.sack_for_metric(self.metric.id),
+                blocking=True, sync=True)
         else:
-            self.chef.process_new_measures(metrics, sync=True)
+            self.chef.refresh_metrics(metrics, timeout=True, sync=True)
```

Comment:
do we have a use case for refreshing multiple metrics at a time? if not, i'd prefer to have this loop and call refresh_metrics rather than modify existing refresh_metrics above just for tests.

Reply:
Other than this, I don't think we have a use case. Though refresh_metrics is not really modified, it's just renamed/merged with process_new_measures.
I can envision a refresh_metrics() for passing `?refresh=true` to the API when doing e.g. dynamic .aggregates